### DISCUSSION TITLE
2005/06/09/%E4%B8%80%E4%BB%BD%E2%80%9C%E8%87%AA%E7%84%B6%E8%BE%A9%E8%AF%81%E6%B3%95%E2%80%9D%E8%AE%BA%E6%96%87

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#35] [Announcements]
# 2005/06/09/%E4%B8%80%E4%BB%BD%E2%80%9C%E8%87%AA%E7%84%B6%E8%BE%A9%E8%AF%81%E6%B3%95%E2%80%9D%E8%AE%BA%E6%96%87

中西方传统自然观的不同，以及对科技发展的影响

https://blog.qizhen.xyz/2005/06/09/%E4%B8%80%E4%BB%BD%E2%80%9C%E8%87%AA%E7%84%B6%E8%BE%A9%E8%AF%81%E6%B3%95%E2%80%9D%E8%AE%BA%E6%96%87

**Comment by ruanqizhen** [comment]
Unknown
2005年06月29日 21:39 编辑
谢谢哦~~~~大方的楼主我正在为自然辩证法的论文头疼呢！！

回复
Qizhen
2005年06月29日 15:41 编辑
本文版权属于全人类，尽管用好了 :)

回复
Unknown
2005年06月29日 12:33 编辑
楼主，拜托你一件事情：这篇文章里的部分资料我有急用，给你打个招呼先~~~~~~~~希望你可以同意，谢谢！！！

回复
王
2005年06月16日 16:16 编辑
上上个礼拜还做了关于欧洲巴洛克园林的报告勒。倒数第二段的后半部分，恩，握个手罢^^^^

回复
Huajun
2005年06月10日 17:33 编辑
早说啊，我就不那么尖刻了。

回复
Qizhen
2005年06月9日 20:01 编辑
我自己写的，水平有限嘛。

回复
Huajun
2005年06月9日 19:30 编辑
这是谁写的，前半部分有几分道理。只是我记得考古好像发现，智人在距今十万年前就已经离开非洲，经过西亚，到达了欧、亚、美洲。P.S.这篇文章从头看到尾，颇有意思！意思不在于文章的内容本身！！作者好像有观点，又不知道或者不想或者不能表述出来，粗粗结了个蛇尾，也许蛇尾都算不上。像是纯属为了应付导师才编出来的结尾。

回复
